Sophos/sophos edr/ingest/parser.yml

```diff
@@ -0,0 +1,83 @@
+name: sophos-edr
+pipeline:
+-   external:
+        name: json.parse-json
+        properties:
+            input_field: '{{original.message}}'
+            output_field: message
+    name: parse_json
+-   external:
+        name: grok.match
+        properties:
+            input_field: '{{parse_json.message.name}}'
+            output_field: message
+            pattern: 'Access was blocked to "%{URL:url}" because of "%{WORD:rulename}".'
+            custom_patterns:
+              URL: '(?:%{URI}|%{URIHOST}|%{URIPATHPARAM}|(?:%{IPORHOST})?:%{POSINT:port})'
+    filter: '{{parse_json.message.type == "Event::Endpoint::WebFilteringBlocked"}}'
+    name: parse_webfiltering
+-   external:
+      name: date.parse
+      properties:
+        input_field: parse_json.message.rt
+        output_field: date
+    name: parse_date
+-   name: set_ecs_fields
+-   name: set_sophos_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+    - set:
```

Comment:
```suggestion
```
I think that you can remove the `- set` without filters here to improve readability

Reply:
Fixed